docs/api/app.md

```diff
@@ -1486,6 +1486,38 @@ This method can only be called after app is ready.
 
 Returns `Promise<string>` - Resolves with the proxy information for `url` that will be used when attempting to make requests using [Net](net.md) in the [utility process](../glossary.md#utility-process).
 
+### `app.setClientCertRequestPasswordHandler(handler)`  _Linux_
+
+* `handler` Function<Promise<string>>
```

Comment:
Based on [setWindowOpenHandler](https://github.com/electron/electron/blob/89ce1cbd395c78d8d5be3457dbbeed176b0a473e/docs/api/web-contents.md?plain=1#L1303-L1305), I think this may need to be escaped.
```suggestion
* `handler` Function\<Promise\<string\>\>
```